HACKING.md

```diff
@@ -80,14 +80,6 @@ to build everything.  Cockpit has a single non-recursive Makefile.  You can
 only run `make` from the top-level and it will always rebuild the whole
 project.
 
-When building from git, you can optionally set the `NO_NPM` environment
-variable to `1` to prevent `autogen.sh` from invoking `npm`.  As a result,
```

Comment:
... and this could be changed to document `$WEBPACK_PACKAGES` then, either for building only one (a common web developer case), or nothing at all (C developer case)